granulate_utils/linux/__init__.py

```diff
@@ -3,12 +3,23 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 import os
+import platform
 from typing import Tuple
 
 
 def get_kernel_release() -> Tuple[int, int]:
```

Comment:
I don't think this change makes sense. `get_kernel_release` is used to check kernel compatibility with features, so it makes no sense to check Windows version here. We should instead `assert platform.system() == "Linux"` and ensure this function isn't used on Windows

Reply:
An assert won't fix the issue I ran into as the os module doesn't have a uname attribute on windows. that's why I check to see if it is running on Linux before calling os.uname(). If I don't have that check, I'll get the following error why PyInstaller attempts to load the granulate_utils/linux module:

 File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "granulate_utils\linux\kernel_messages.py", line 30, in <module>
  File "granulate_utils\linux\__init__.py", line 15, in get_kernel_release
**AttributeError: module 'os' has no attribute 'uname'**
[25204] Failed to execute script 'pyi_build' due to unhandled exception!

Reply:
Hmm. Logically `kernel_messags.py` should not be imported at all when we run the Windows version. Hence I see no reason to make this change. Let's remove this change and try to avoid the import in gProfiler's/PyInstaller's side?